Dockerfile

```
@@ -0,0 +1,24 @@
+# Miden-node Dockerfile
+
+#### Setup image builder
+FROM rust:1.75.0-bullseye AS builder
+
+# Install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y gcc llvm clang bindgen pkg-config
```

Let's merge these two layers so that we use less disk:

```suggestion
RUN apt-get update && apt-get -y upgrade && apt-get install -y gcc llvm clang bindgen pkg-config
```